### PR TITLE
🐛 Fix race condition for `signals`

### LINKED ISSUE #21186: Error: b.signals is not a function. (In 'b.signals()', 'b.signals' is undefined)
Error: b.signals is not a function. (In 'b.signals()', 'b.signals' is undefined)
at (https://raw.githubusercontent.com/ampproject/amphtml/1902261915030/extensions/amp-auto-lightbox/0.1/amp-auto-lightbox.js:279)
at (https://raw.githubusercontent.com/ampproject/amphtml/1902261915030/extensions/amp-auto-lightbox/0.1/amp-auto-lightbox.js:473)
at ampdoc (https://raw.githubusercontent.com/ampproject/amphtml/1902261915030/extensions/amp-auto-lightbox/0.1/amp-auto-lightbox.js:498)
at ampdoc (https://raw.githubusercontent.com/ampproject/amphtml/1902261915030/extensions/amp-auto-lightbox/0.1/amp-auto-lightbox.js:507)

go/amp-error/CO7PhqTDidyMgQE
error from canary, looks new to me.

### PR DESCRIPTION
**Pull request by alanorozco**
Fixes #21186.

### CODE REVIEW

**Comment by alanorozco** [comment]
@rsimha bundle-bot added you due to a check failure :(

File: extensions/amp-auto-lightbox/0.1/amp-auto-lightbox.js [hunk 1]
```diff
@@ -273,10 +273,11 @@ function candidateSelector(tagName) {
 
 /**
  * @param {!Element} element
- * @return {!Promise<!Element>}
+ * @return {!Promise}
  */
 function whenLoaded(element) {
-  return element.signals().whenSignal(CommonSignals.LOAD_END);
```
Comment (jridgewell) [inline extensions/amp-auto-lightbox/0.1/amp-auto-lightbox.js]: Hmm, this is a bug with our externs. `signals` isn't on `Element`, only an `AmpElement`. We'll likely catch other bugs.

**Comment by rsimha** [comment]
@alanorozco Rerunning the bundle-size [job](https://travis-ci.org/ampproject/amphtml/jobs/501608124) should fix it. I've done it for you. (Root cause of flakiness has been fixed this morning.)

**Comment by alanorozco** [comment]
@rsimha Yeah, just wanted to make sure the issue was passed on 👍

**Pull request merged**